pkg/cmd/ssh/ssh_suite_test.go

```diff
@@ -25,6 +26,8 @@ func init() {
 }
 
 func TestCommand(t *testing.T) {
+	SetDefaultEventuallyTimeout(5 * time.Second)
+	SetDefaultEventuallyPollingInterval(200 * time.Millisecond)
```

Comment:
```suggestion
```
remove overwriting the default polling interval if there is no real requirement for it.

Reply:
Sure, done with https://github.com/gardener/gardenctl-v2/pull/348/commits/16b5084bfe2a8b960e04d42d5089fd146839eda5